docs/security-considerations.rst

```diff
@@ -116,10 +116,27 @@ outlined further below:
         }
     }
 
-Note that re-entrancy is not only an effect of Ether transfer but of any
-function call on another contract. Furthermore, you also have to take
-multi-contract situations into account. A called contract could modify the
-state of another contract you depend on.
+The Checks-Effects-Interactions pattern ensures that all code paths through a contract complete all required verification
+of supplied parameters before modifying contract state (Checks), then makes any changes to contract state (Effects),
+then only makes calls to functions in other contracts *after* all planned changes to contract state have been written to
+storage (Interactions). This is the only completely foolproof way to prevent *reentrancy attacks*, where a call to an
```

Comment:
While I may be in favor of a little section on CEI, I disagree with this particular wording. This is **not** the only completely foolproof way to prevent reentrancy. Custom logic is often enough to guarantee that reentrancy is not harmful.